tools/toolchain/scripts/install_sirius.sh

```diff
@@ -127,19 +127,14 @@ case "$with_sirius" in
                   -DCMAKE_CXX_COMPILER=mpic++ \
                   -DCMAKE_C_COMPILER=mpicc ${COMPILATION_OPTIONS} .."
 
-             cmake -DCMAKE_INSTALL_PREFIX=${pkg_install_dir} \
+            cmake -DCMAKE_INSTALL_PREFIX=${pkg_install_dir} \
                   -DCMAKE_CXXFLAGS_RELEASE="${SIRIUS_OPT}" \
                   -DCMAKE_CXX_FLAGS_RELWITHDEBINFO="${SIRIUS_DBG}" \
                   -DCMAKE_CXX_COMPILER=mpic++ \
                   -DCMAKE_C_COMPILER=mpicc \
-                  ${COMPILATION_OPTIONS} .. > compile.log 2>&1
-             make -j $NPROCS -C src >> compile.log 2>&1
-
-            install -d ${pkg_install_dir}/include >> install.log 2>&1
-            install -d ${pkg_install_dir}/lib >> install.log 2>&1
-            cp -R ../src/* ${pkg_install_dir}/include >> install.log 2>&1
-            install -m 644 src/*.a ${pkg_install_dir}/lib >> install.log 2>&1
-            install -m 644 src/mod_files/*.mod ${pkg_install_dir}/include >> install.log 2>&1
+                  ${COMPILATION_OPTIONS} -DCMAKE_INSTALL_PREFIX=${pkg_install_dir} .. #> compile.log 2>&1
+            make -j $NPROCS #>> compile.log 2>&1
+	        make install #>> compile.log 2>&1
```

Comment:
I guess these redirects are going to change again before the merge?

Reply:
Yes. I try to know why it does not pass your ci while it compile fine on my desktop. Not having access to the log file is a pain.

Reply:
The [log file problem](https://github.com/cp2k/cp2k-ci/issues/48) should be fixed now. Alternatively, you can always run `docker build` locally to reproduce what the CI is doing.

Reply:
Instead of adding/removing redirects we could write a small helper function that runs a command but only prints its output when the command has failed. See also https://github.com/cp2k/cp2k/issues/433.

Reply:
thanks for the mod.